fdbserver/workloads/SnapTest.actor.cpp

```diff
@@ -0,0 +1,618 @@
+#include "fdbserver/Status.h"
+#include "flow/actorcompiler.h"
+#include "fdbrpc/ContinuousSample.h"
+#include "fdbclient/NativeAPI.actor.h"
+#include "fdbclient/ManagementAPI.actor.h"
+#include "fdbserver/TesterInterface.actor.h"
+#include "fdbserver/WorkerInterface.actor.h"
+#include "workloads.actor.h"
+#include "BulkSetup.actor.h"
+#include "fdbserver/ClusterRecruitmentInterface.h"
+#include "fdbclient/ReadYourWrites.h"
+#include "fdbmonitor/SimpleIni.h"
+#include <boost/lexical_cast.hpp>
+
+#undef FLOW_ACOMPILER_STATE
+#define FLOW_ACOMPILER_STATE 1
+
+void getVersionAndnumTags(TraceEventFields md, Version& version, int& numTags) {
+	version = -1;
+	numTags = -1;
+
+	sscanf(md.getValue("Version").c_str(), "%lld", &version);
+	sscanf(md.getValue("NumTags").c_str(), "%d:%d", &numTags);
+}
+
+void getTagAndDurableVersion(TraceEventFields md, Version version, Tag& tag, Version& durableVersion) {
+	Version verifyVersion;
+	durableVersion = -1;
+
+	int tagLocality;
+	int tagId;
+	sscanf(md.getValue("Version").c_str(), "%lld", &verifyVersion);
+	sscanf(md.getValue("Tag").c_str(), "%d:%d", &tagLocality, &tagId);
+	tag.locality = tagLocality;
+	tag.id = tagId;
+	sscanf(md.getValue("DurableVersion").c_str(), "%lld", &durableVersion);
```

Comment:
boost::lexical_cast is the properly c++ way to do string->int conversions

Reply:
there is inconsistent use of this style in the code, but switched it to ::lexical_cast.